scalastyle-config.xml

```diff
@@ -0,0 +1,156 @@
+		<parameters>
+			<parameter name="ignore">-1,0,1,2,3</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.NoWhitespaceBeforeLeftBracketChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.NoWhitespaceAfterLeftBracketChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.ReturnChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.NullChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.NoCloneChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.NoFinalizeChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.CovariantEqualsChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.StructuralTypeChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.file.RegexChecker" enabled="true">
+		<parameters>
+			<parameter name="regex">println</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.NumberOfTypesChecker" enabled="true">
+		<parameters>
+			<parameter name="maxTypes">30</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.CyclomaticComplexityChecker" enabled="true">
+		<parameters>
+			<parameter name="maximum">10</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.UppercaseLChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.SimplifyBooleanExpressionChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.scalariform.IfBraceChecker" enabled="true">
+		<parameters>
+			<parameter name="singleLineAllowed">true</parameter>
+			<parameter name="doubleLineAllowed">false</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.MethodLengthChecker" enabled="true">
+		<parameters>
+			<parameter name="maxLength">50</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.MethodNamesChecker" enabled="true">
+		<parameters>
+			<parameter name="regex">^[a-z][A-Za-z0-9]*$</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.NumberOfMethodsInTypeChecker" enabled="true">
+		<parameters>
+			<parameter name="maxMethods">30</parameter>
+		</parameters>
+	</check>
+	<check level="warning" class="org.scalastyle.scalariform.PublicMethodsHaveTypeChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.file.NewLineAtEofChecker" enabled="true"/>
+	<check level="warning" class="org.scalastyle.file.NoNewLineAtEofChecker" enabled="false"/>
+	<check level="warning" class="org.scalastyle.scalariform.WhileChecker" enabled="false"/>
+	<check level="warning" class="org.scalastyle.scalariform.VarFieldChecker" enabled="true"/>
```

Comment:
I'd like to avoid it for fields, I can live with local `var`s. It's just a warning, not an error, won't fail the build.

Reply:
I don't want to fail on warning, not just because there's a lot that needs refactoring, but because I acknowledge there are some valid cases. A warning is supposed to make you think twice about what you're doing. In my opinion `var` fields can be more pernicious.

Reply:
Once warnings reach a larger amount they stop to serve the purpose (unless some tool is used to empathize the new ones, of course).

Reply:
Fair enough